partials/about.html

```diff
@@ -5,7 +5,7 @@
                     <div class="col-xs-12 col-sm-6 col-sm-offset-3">
                     	<div class="supporting-section-title">Method</div>
                     	<div class="supporting-section-body">
-                        	Financial independence is the point at which passive income safely exceeds expenses. "Safely" is determined by <a href="http://en.wikipedia.org/wiki/Trinity_study">the 4% safe withdrawal rate,</a> which is the maximum rate at which you can spend your savings such that you don’t run out in your lifetime. With your assets invested in diversified stock market funds which historically gain on average 7-12% annually, one can safely live off passive income forever. <a href="http://www.mrmoneymustache.com/2012/05/29/how-much-do-i-need-for-retirement/">Learn more<a/>
+                        	Financial independence is the point at which passive income safely exceeds expenses. "Safely" is determined by <a href="http://en.wikipedia.org/wiki/Trinity_study">the 4% safe withdrawal rate,</a> which is the maximum rate at which you can spend your savings such that you don’t run out in your lifetime. With your assets invested in diversified stock market funds which historically gain on average 7-12% annually, one can safely live off passive income forever.
```

Comment:
I put it back in. The original thought was, users may not expect to be taken to a different site. But, for now I don't think we need to write more content, so I'll just do a target="_blank"